_activities/pyhep.md

```diff
@@ -0,0 +1,46 @@
+---
+title: PyHEP - "Python in HEP" Series of Workshops
+author: "Eduardo Rodrigues"
+layout: default
+redirect_from:
+  - /pyhep.html
```

Comment:
Surely we don't need this?

Reply:
Ah, it's also causing the spurious error "External link http://www.hepsoftwarefoundation.org/activities/pyhep.html failed: 404 No error" as the redirect gets written with the site name in it.

Reply:
OK, these lines are just a (mutatis mutandis) copy-and-paste from other .md pages. But will remove them.